src/browsers/tob/shared/components/ManhattanPlotNew.js

```diff
@@ -0,0 +1,362 @@
+              <defs>
+                <clipPath id="clipManhattanPlot">
+                  <rect width={innerWidth} height={innerHeight} fill="none" pointerEvents="all" />
+                </clipPath>
+              </defs>
+              {/* <rect width={innerWidth} height={innerHeight} fill="none" stroke="black" /> */}
+              <g transform={`translate(0, ${innerHeight})`}>
+                <line x2={`${innerWidth}`} stroke="black" />
+
+                {xScaleLocal.ticks(6).map((tick) => (
+                  <g key={tick} transform={`translate(${xScaleLocal(tick)}, 0)`}>
+                    <text style={{ textAnchor: 'middle' }} dy=".71em" y={9}>
+                      {numberWithCommas(tick)}
+                    </text>
+                    <line y2={6} stroke="black" />
+                  </g>
+                ))}
+              </g>
+              <line y2={`${innerHeight}`} stroke="black" />
+              {yScaleLocal.ticks(4).map((tick) => (
+                <g key={tick} transform={`translate(0, ${yScaleLocal(tick)})`}>
+                  <text key={tick} style={{ textAnchor: 'end' }} x={-6} dy=".32em">
+                    {tick}
+                  </text>
+                  <line x2={-3} stroke="black" />
+                  <line
+                    x2={innerWidth}
+                    stroke={yScaleLocal(tick) === innerHeight ? 'none' : 'lightgrey'}
+                  />
+                </g>
+              ))}
+              <g ref={brushRef} />
+              <g clipPath="url(#clipManhattanPlot)">
+                {data.map((d) => (
+                  <React.Fragment key={`${accessors.x(d)},${accessors.y(d)},${accessors.color(d)}`}>
+                    {(accessors.isSelected(d) || accessors.isReference(d)) && (
+                      <line
+                        key={`${accessors.x(d)},${accessors.y(d)},${accessors.color(
+                          d
+                        )}SelectedLine`}
+                        x1={xScaleLocal(accessors.x(d))}
+                        x2={xScaleLocal(accessors.x(d))}
+                        y1={0}
+                        y2={innerHeight}
+                        strokeDasharray={12}
+                        stroke="black"
+                        opacity="0.5"
+                      />
+                    )}
+                    {accessors.isReference(d) ? (
+                      <g
+                        transform={`translate(${xScaleLocal(accessors.x(d))}, ${yScaleLocal(
+                          accessors.y(d)
+                        )}),rotate(45)`}
+                      >
+                        <rect
+                          key={`${accessors.x(d)},${accessors.y(d)},${accessors.color(d)}`}
+                          width={10}
+                          height={10}
+                          fill={defaultCellTypeColors()[accessors.color(d)]}
+                          onClick={onClick}
+                          // onMouseOver={() => onMouseOver()}
```

Comment:
Missing opacity property derived from `accessors?.opacity(d)` which should be set to 0 if undefined.